service/src/main/java/greencity/service/EmailServiceImpl.java

```diff
@@ -241,7 +241,11 @@ private void sendEmail(String receiverEmail, String subject, String content) {
     public void sendHabitNotification(String name, String email) {
         String subject = "Notification about not marked habits";
         String content = "Dear " + name + ", you haven't marked any habit during last 3 days";
-        sendEmail(email, subject, content);
+        if (!userRepo.existsUserByEmail(email)) {
+            throw new NotFoundException(ErrorMessage.USER_NOT_FOUND_BY_EMAIL + email);
+        }else {
+            sendEmail(email, subject, content);
+        }
```

Comment:
Else is not necessary:
`if (!userRepo.existsUserByEmail(email)) {
            throw new NotFoundException(ErrorMessage.USER_NOT_FOUND_BY_EMAIL + email);
        }
        sendEmail(email, subject, content);`

Reply:
Thanks for the advice, I tried to follow the code that was written before me...
Anyways, I'll clean that "else" in my next commit, thank you for review!